src/dungeon.cc

```diff
@@ -4185,12 +4158,6 @@ static void process_command(void)
 			do_cmd_macro_recorder();
 			break;
 		}
-	case CMD_BLUNDER:
-		{
-			if (do_control_walk()) break;
-			do_cmd_walk(always_pickup, FALSE);
```

Comment:
Well, it's easy to check if it's ever called with anything but FALSE.

It's also fine to do this in a follow-up PR just to make it simpler to review.